include/dfmessages/Types.hpp

```diff
@@ -40,6 +42,18 @@ using timestamp_t = dataformats::timestamp_t;           ///< Copy dataformats::t
 using timestamp_diff_t = dataformats::timestamp_diff_t;           ///< Copy dataformats::timestamp_diff_t
 
 using trigger_type_t = dataformats::trigger_type_t; ///< Copy dataformats::trigger_type_t
+
+/**
+ * @brief Which type of readout to use for TriggerDecision and DataRequest
+*/
+enum class ReadoutType : uint16_t
```

Comment:
Whoever maintains trigger decision generator modules might want to comment here if we need ToString/From string operations. 
I'm not one of those people, so I'll defer and approve.